From a8d85949a068ee6cd9a2f923cd039047993f239e Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 3 Apr 2017 17:11:32 -0700
Subject: [PATCH] include required system header files for fd_set and makedev

fd_set comes from sys/select.h
makedev macro is defined in sys/sysmacros.h

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 uml_net/ethertap.c | 1 +
 uml_net/host.c     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/uml_net/ethertap.c b/uml_net/ethertap.c
index ddd41f2..57f7119 100644
--- a/uml_net/ethertap.c
+++ b/uml_net/ethertap.c
@@ -11,6 +11,7 @@
 #include <sys/signal.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
+#include <sys/select.h>
 #include <sys/ioctl.h>
 #include <net/if.h>
 #include "host.h"
diff --git a/uml_net/host.c b/uml_net/host.c
index 65c20df..fc04cd4 100644
--- a/uml_net/host.c
+++ b/uml_net/host.c
@@ -10,6 +10,7 @@
 #include <ctype.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include "output.h"
 #include "host.h"
 
-- 
2.12.2

